marlowe/src/Language/Marlowe/Core/V1/Semantics.hs

```diff
@@ -104,7 +104,7 @@ import Text.PrettyPrint.Leijen (comma, hang, lbrace, line, rbrace, space, text,
     when positive balances are payed out on contract closure.
 -}
 data Payment = Payment AccountId Payee Money
-  deriving stock (Haskell.Show)
+  deriving stock (Haskell.Show, Haskell.Eq)
```

Comment:
This induced a compilation failure for `exe:marlowe-test`. Here is the patch that fixes it:
```
diff --git a/marlowe/test/Spec/Marlowe/Orphans.hs b/marlowe/test/Spec/Marlowe/Orphans.hs
index 7a9b351db..23ee77c1d 100644
--- a/marlowe/test/Spec/Marlowe/Orphans.hs
+++ b/marlowe/test/Spec/Marlowe/Orphans.hs
@@ -10,9 +10,6 @@ module Spec.Marlowe.Orphans (
 import Language.Marlowe.Core.V1.Semantics
 
 
-deriving instance Eq Payment
-
-
 deriving instance Eq ApplyWarning
```

I recommend running `cabal build all` and `cabal test all` for PRs.

Reply:
Thanks for that! I wonder why this was defined here instead of in the original module - is it to save space in the compiled plutus code? Also, RE:

> I recommend running `cabal build all` and `cabal test all` for PRs.

But that's why we have a build server, so I don't have to do this locally :smirk:

Reply:
I'll actually just revert this change, as I don't need it any longer